libnmstate/schemas/operational-state.yaml

```diff
@@ -24,6 +24,14 @@ properties:
             - "$ref": "#/definitions/interface-ovs-interface/rw"
             - "$ref": "#/definitions/interface-dummy/rw"
             - "$ref": "#/definitions/interface-vlan/rw"
+  routing:
+    type: object
+    properties:
+      ribs:
```

Comment:
Can we use singular (for `rib`, `route` and so on..)?

Reply:
Changed.